.github/workflows/ci.yml

```
@@ -633,3 +633,7 @@ jobs:
         run: |
           cd babel-test262-runner
           node lib/compare-results ~/test262-main.tap ../test262.tap | tee ~/diff.tap
+          exit 1 | tee test
```

```suggestion
          node lib/compare-results ~/test262-main.tap ../test262.tap | tee ~/diff.tap
```
This was just to test if it works